open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md

```diff
@@ -150,10 +150,10 @@ Now, it's time to _pickle_ your model! You can do that in a few lines of code. O
 ```{code-cell}
 import pickle
 model_filename = 'ufo-model.pkl'
-pickle.dump(model, open(model_filename,'wb'))
+pickle.dump(model, open(model_filename, 'wb'))
 
-model = pickle.load(open('ufo-model.pkl','rb'))
-print(model.predict(pd.DataFrame([[50,44,-12]])))
+model = pickle.load(open('ufo-model.pkl', 'rb'))
```

Comment:
Is this the right path? Can you make sure your code is successfully built?

Reply:
This change has been implemented in the latest version of open-academy

Reply:
The path has been modified to ../assets/pickle/ufo-model.pkl，so I didn't make any more changes